qiskit_ibm_runtime/utils/utils.py

```diff
@@ -73,10 +73,11 @@ def _is_isa_circuit_helper(circuit: QuantumCircuit, target: Target, qubit_map: D
         # errors (beyond pi/2 does not trigger an error down the stack, only may become less
         # accurate).
         if name == "rzz" and (
-            instruction.params[0] < 0.0 or instruction.params[0] > 1.001 * np.pi / 2
```

Comment:
Out of scope, but a question for @yaelbh when she returns: what if `params[0]` is not numeric, but a `Parameter`?

Reply:
I'll check. Marking this conversation as resolved, since it won't let merge the branch otherwise.